.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,73 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-h2-image:
+    environment: dockerhub-publish
+    name: Push h2 db Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
where do this come from? which credentials are being used? Do we need to add them to github secrets?

Reply:
Added comments to the top explaining this.

Reply:
Added instructions to the top of the yml file